### ISSUE TITLE
Mesa fails to build

### ISSUE THREAD

**Issue by JL2210** [skycocker/chromebrew#2500]
```
  CC       gallivm/lp_bld_intr.lo
  CC       gallivm/lp_bld_logic.lo
  CXX      gallivm/lp_bld_misc.lo
  CC       gallivm/lp_bld_pack.lo
  CC       gallivm/lp_bld_printf.lo
  CC       gallivm/lp_bld_quad.lo
  CC       gallivm/lp_bld_sample_aos.lo
  CC       gallivm/lp_bld_sample.lo
gallivm/lp_bld_misc.cpp: In function ‘LLVMOpaqueBuilder* lp_create_builder(LLVMContextRef, lp_float_mode)’:
gallivm/lp_bld_misc.cpp:833:13: error: ‘class llvm::FastMathFlags’ has no member named ‘setUnsafeAlgebra’
       flags.setUnsafeAlgebra();
             ^~~~~~~~~~~~~~~~
gallivm/lp_bld_sample.c: In function ‘lp_build_rho’:
gallivm/lp_bld_sample.c:305:21: warning: ‘ddx[1]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
          rho_xvec = lp_build_add(coord_bld, ddx[0], ddx[1]);
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gallivm/lp_bld_sample.c:305:21: warning: ‘ddx[0]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
gallivm/lp_bld_sample.c:306:21: warning: ‘ddy[1]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
          rho_yvec = lp_build_add(coord_bld, ddy[0], ddy[1]);
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gallivm/lp_bld_sample.c:306:21: warning: ‘ddy[0]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
gallivm/lp_bld_sample.c:315:14: warning: ‘ddmax[0]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
          rho = ddmax[0];
          ~~~~^~~~~~~~~~
  CC       gallivm/lp_bld_sample_soa.lo
make[4]: *** [Makefile:2538: gallivm/lp_bld_misc.lo] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: Leaving directory '/usr/local/tmp/crew/mesa-17.3.6.tar.xz.dir/mesa-17.3.6/src/gallium/auxiliary'
make[3]: *** [Makefile:610: all-recursive] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/mesa-17.3.6.tar.xz.dir/mesa-17.3.6/src/gallium'
make[2]: *** [Makefile:872: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/mesa-17.3.6.tar.xz.dir/mesa-17.3.6/src'
make[1]: *** [Makefile:657: all] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/mesa-17.3.6.tar.xz.dir/mesa-17.3.6/src'
make: *** [Makefile:659: all-recursive] Error 1
mesa failed to install: `make V=0 -j2` exited with 2
```
Command: `crew install -s mesa`

Notes: I also had to edit /usr/local/etc/crew/packages and remove radeon

**Comment by JL2210** [comment]
I believe this isn't necessarily a GUI issue, more like a compilation issue.

**Comment by uberhacker** [comment]
@JL2210: You might try updating.  See https://mesa.freedesktop.org/archive/mesa-18.1.5.tar.xz.

**Comment by skiman6010** [comment]
Mesa is part of the gui work so it is a gui related issue.

**Comment by JL2210** [comment]
@uberhacker: Thanks, that worked. I'll add it to my PR when I get done with the `a` packages.

**Comment by JL2210** [comment]
glibmm and atkmm are unupdateable, can you remove them from the list?

**Comment by uberhacker** [comment]
The only way we can remove anything from the list if if you submit PRs with valid reasons for removal.

**Comment by JL2210** [comment]
A pull request on an issue?

**Comment by JL2210** [comment]
Here's a valid reason: They fail to compile.

**Comment by uberhacker** [comment]
Fixed in #2553.
